crates/typed-store/src/rocks/mod.rs

```diff
@@ -2428,7 +2428,14 @@ pub fn default_db_options() -> DBOptions {
         read_size_from_env(ENV_VAR_DB_WAL_SIZE).unwrap_or(DEFAULT_DB_WAL_SIZE) as u64 * 1024 * 1024,
     );
 
+    // Num threads for compaction and flush.
     opt.increase_parallelism(4);
+
+    // Disable write stalling and stopping based on pending compaction bytes.
+    // Throttling writes in Sui does not reduce the incoming data throughput most of the time,
+    // especially on the larger column families that do not get deleted per epoch.
+    opt.set_hard_pending_compaction_bytes_limit(0);
```

Comment:
do you want to adjust soft_pending_compaction_bytes_limit as well?

Reply:
Yes. Somehow that change was lost.